### ISSUE TITLE
Request for Custom Dataset Training Guidelines

### ISSUE THREAD

**Issue by ys31jp** [weiguangzhao/PBNet#5]
Hi, thank you for the work. I am interested in training custom datasets using your method. 
Would there be provide sooner?
Thank you.

**Comment by weiguangzhao** [comment]
Thanks for your attention. We will provide custom data training guidance as soon as possible. In general, we may need approximately **one month** for debugging.

**Comment by ys31jp** [comment]
Thank you very much for your prompt response.
At the meantime, would there be any recommend way for me to run training before releasing of the guideline?
like the dataset would be in scannet format or s3dis format.

**Comment by weiguangzhao** [comment]
There are three points you may need to pay attention to:

1.Convert all data to the npy format, including XYZ [N, 3]; RGB [N, 3]; Semantic Label [N]; Instance Label [N]; and set the labels for invalid or noisy points to -100. For this part, you need to refine the dataset.py file.

2.Finetuning the hyperparameter raduis and min_pts in the config.py&config_test.py.  We recommend standardizing the coordinate data to the meter level, then setting the radius to 0.04, and min_pts to 1.5 times the original point density.

3. Finally, you need to modify the number of semantic categories 

Feel free to ask if you have any questions.

**Comment by zhou991119** [comment]
Could you please provide the code for Single_card_training? Thank you
